tests/test_model_specs.py

```diff
@@ -97,6 +97,9 @@ def validate_output(self, spec, key, parent_type=None):
             # if parent_type is None:  # all top-level args must have these attrs
             #     for attr in ['about']:
             #         self.assertIn(attr, spec)
+            if 'docs' in spec and spec['docs'] == False:
```

Comment:
Sorry, what's the `docs` key in a spec?

Reply:
Oops, I was using `"docs": False` as a placeholder before I filled in the outputs that were previously not documented. Removed this!